frontend/src/rides.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from "react";
+
+function RideModal({ ride, onClose }) {
+    if (!ride) return null;
+
+    // Definer styles osv, for at undgå at bruge App.css fordi der blev snakket om den skulle slettes
+    const modalStyle = {
```

Comment:
This could be done with tailwind classes instead. No need to update it now just fyi for the future

Reply:
Seems like github didn't like me marking multiple lines. I meant the CSS classes in rides.jsx

Reply:
@roastedtoastyt